src/encord_active/cli/utils/decorators.py

```diff
@@ -32,78 +32,94 @@ def find_child_projects(target: Path) -> List[Path]:
-            return fn(*args, **kwargs)
+                kwargs["target"] = parent_target
+                return fn(*args, **kwargs)
 
-        child_projects = find_child_projects(target)
-        if child_projects:
-            from InquirerPy import inquirer as i
-            from InquirerPy.base.control import Choice
+            child_projects = find_child_projects(target)
 
-            rich.print(
-                Panel(
-                    """
-The specified directory is not an Encord Project. 
-Encord Active found the following projects to choose from.
+            if not child_projects:
+                rich.print(
+                    Panel(
+                        f"""
+        Couldn't find a project at: [purple]{kwargs['target']}[/purple].
+        Either [blue]`cd`[/blue] into a directory containing a project or specify the path with the `--target` option.
+
+        :bulb: hint: By default, projects are stored in the current working directory when you run the [blue]init[/blue], [blue]import[/blue], and [blue]download[/blue] commands.
                     """,
-                    title="🕵️  Select a project 🕵️",
-                    expand=False,
-                    style="blue",
+                        title=":exclamation: Couldn't find a project :exclamation:",
+                        style="yellow",
+                        expand=False,
+                    )
                 )
-            )
-            choices = list(map(lambda p: Choice(p, name=p.name), child_projects))
-            choosen_target = i.fuzzy(
-                message="Choose a project",
-                choices=choices,
-                vi_mode=True,
-                instruction="Type a search string or use arrow keys and hit [enter]. Hit [ctrl-c] to abort.",
-            ).execute()
-            if choosen_target:
-                kwargs["target"] = choosen_target
-                return fn(*args, **kwargs)
-            rich.print("No project was selected. Aborting.")
-            raise typer.Exit()
 
-        rich.print(
-            Panel(
-                f"""
-Couldn't find a project at: [purple]{kwargs['target']}[/purple].
-Either [blue]`cd`[/blue] into a directory containing a project or specify the path with the `--target` option.
+                raise typer.Exit()
 
-:bulb: hint: By default, projects are stored in the current working directory when you run the [blue]init[/blue], [blue]import[/blue], and [blue]download[/blue] commands.
-            """,
-                title=":exclamation: Couldn't find a project :exclamation:",
-                style="yellow",
-                expand=False,
-            )
-        )
+            if not allow_multi:
+                choosen_target = choose_project(child_projects)
+                if not choosen_target:
+                    rich.print("No project was selected. Aborting.")
+                    raise typer.Exit()
 
-        raise typer.Exit()
+                kwargs["target"] = choosen_target
```

Comment:
I am not sure that this is something that we would want to do as a side-effect in this function? Shouldn't it be in the visualize definition instead?